tests/default/indices/msearch.yaml

```diff
@@ -0,0 +1,124 @@
+$schema: ../../../json_schemas/test_story.schema.yaml
+
+description: Test endpoints relevant to the lifecycle of an index, including multi-get and multi-search operations.
+epilogues:
+  - path: /books
+    method: DELETE
+    status: [200, 404]
+  - path: /games
+    method: DELETE
+    status: [200, 404]
+chapters:
+  - synopsis: Create an index named `books` with mappings and settings.
+    path: /{index}
+    method: PUT
+    parameters:
+      index: books
+    request:
+      payload:
+        mappings:
+          properties:
+            name:
+              type: keyword
+            age:
+              type: integer
+        settings:
+          number_of_shards: 5
+          number_of_replicas: 2
```

Comment:
I don't think number of shards and replicas is material for this test, remove. If msearch works the same without a mapping, as well.